database/db_tools.py

```diff
@@ -144,18 +144,19 @@ def log_unmapped_handles(site_username_tuple_list: List[Tuple[str, str]]) -> Non
 
         log_unmapped_handles(site_handle_tuple_list)
 
-    logging.info('Mapped ')
+    info('Mapped usernames to SRNs')
 
 
 def remove_unmapped_handles_from_rank_file(file_name: str) -> None:
     """
     Removes unmapped handles from outdated rank files
     to reduce space and time it takes for the script to run
     """
-    with open(join(CONTEST_RANKS_DIR, file_name), 'r') as rank_file:
+    with open(file_name, 'r') as rank_file:
         input_data: str = rank_file.read()
-
-    with open(join(CONTEST_RANKS_DIR, file_name), 'w') as rank_file:
+    #print(len(input_data))
```

Comment:
Remove this comment, and what's the purpose of `count`?

Reply:
nothing, twas for debugging; has been removed